src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -1293,34 +1310,47 @@ private void AbortStreams(int lastValidStream, Exception abortException)
 
                         _httpStreams.Remove(kvp.Value.StreamId);
                     }
-                    else if (NetEventSource.IsEnabled)
+                    else
                     {
-                        Trace($"Found {nameof(streamId)} {streamId} <= {lastValidStream}.");
+						if (NetEventSource.IsEnabled)
+	                    {
+	                        Trace($"Found {nameof(streamId)} {streamId} <= {lastValidStream}.");
+	                    }
```

Comment:
Nit: our style for tracing is to do it all one one-line, treating the "if (NetEventSource.IsEnabled)" as a prefix we'd like to be able to hide but can't for perf reasons, e.g.
```C#
if (NetEventSource.IsEnabled) Trace($"Found {nameof(streamId)} {streamId} <= {lastValidStream}.");
```
I didn't previously do this when it would flow as part if the cascading if/else if, but now that you've made it nested, it'd be good to change it back.